src/App.js

```diff
@@ -1,29 +1,15 @@
-import logo from './logo.svg';
-import './App.css';
+import React, { Component } from 'react';
+import Calculator from './components/calculator';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+// eslint-disable-next-line react/prefer-stateless-function
+class App extends Component {
+  render() {
+    return (
+      <div className="App">
+        <Calculator />
+      </div>
+    );
+  }
```

Comment:
- [ ] Great work on implementing the requested change by the previous reviewer, however you have also introduced a `disable linter` comment which is not a best coding practice. Kinly remove it as linters help us to write clean industry standard code.

Reply:
@Munsa1 I tried everything to remove the error but it is not removed so stack flow showed me this solution